gcloud/bigtable/_testing.py

```diff
@@ -41,17 +31,16 @@ def __getattr__(self, name):
 class _MethodMock(object):
     """Mock for API method attached to a gRPC stub.
 
-    In the beta implementation, these are of type.
-    :class:`grpc.framework.crust.implementations._UnaryUnaryMultiCallable`
+    These are of type :class:`grpc._channel._UnaryUnaryMultiCallable`.
```

Comment:
I never looked at `__bases__` I just did a `type(DatastoreStub().Commit)` and wrote that. I don't quite understand the value of handing off private instances and telling people to care about the public base class.